Extension/lib/content-script/assistant/js/assistant.js

```diff
@@ -813,7 +757,8 @@ var Adguard = function () {
 	};
 
 	var hideElement = function () {
-		self.selector.removeBorders();
+		AdguardSelectorLib.reset();
+
 		var selector = getSelector();
 		var style = document.createElement("style");

```

Comment:
In case of block by url, getSelector() returns url rule, not css rule and element wouldn't be hidden.